CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+# Contributing to the Firebase Analytics Webview
```

Comment:
I guess this is inspired by Firebase project. :)
We should update it with our own.

Reply:
Yup, hard to deny where I was inspired from now 😅 Fixed, thanks!